onnxruntime/core/providers/openvino/onnx_ctx_model_helper.cc

```diff
@@ -77,7 +77,7 @@ Status EPCtxHandler::ExportEPCtxModel(const GraphViewer& graph_viewer,
   model_proto->set_ir_version(ONNX_NAMESPACE::Version::IR_VERSION);
 
   // Finally, dump the model
-  std::ofstream dump(graph_name + "-ov_" + device_type + "_blob.onnx",
+  std::ofstream dump(graph_name,
```

Comment:
Optional: Its good to handle the I/O operations like below -

```suggestion
  std::ofstream dump_epctx_onnx_model(graph_name, std::ios::out | std::ios::trunc | std::ios::binary);
  if (!dump_epctx_onnx_model) { 
     ORT_THROW("Unable to open file for writing."); 
    }
```